src/main.go

```diff
@@ -1009,6 +1067,53 @@ func consumeError() []error {
 func main() {
 	flag.Parse()
 	tests := flag.Args()
+	startTime := time.Now()
+
+	if xmlPath != "" {
+		_, err := os.Stat(xmlPath)
+		if err == nil {
+			err = os.Remove(xmlPath)
+			if err != nil {
+				log.Errorf("drop previous junit file fail:", err)
+				return
+			}
+		}
+
+		xmlFile, err = os.Create(xmlPath)
+		if err != nil {
+			log.Errorf("create junit file fail:", err)
+			return
+		}
+		xmlFile, err = os.OpenFile(xmlPath, os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0600)
+		if err != nil {
+			log.Errorf("open junit file fail:", err)
+			return
```

Comment:
IMO a non-zero return value is still needed, when there's an error. The CI of TiDB may relies on it.